docs/app/docs/devbox_examples/databases/mysql.md

```diff
@@ -0,0 +1,71 @@
+---
+title: MySQL
+---
+MySQL can be automatically configured for your dev environment by Devbox via the built-in MySQL Plugin. This plugin will activate automatically when you install MySQL using `devbox add mysql80` or `devbox add mysql57`.
```

Comment:
can you add in parentheses "(for mysql v8.0)" and "(for mysql v5.7)" to make it clearer what those numbers are